exhibits-solr-conf.gemspec

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'exhibits-solr-conf/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'exhibits-solr-conf'
```

Comment:
I'd argue that following conventions is generally useful. It certainly makes it easier to look at a project and figure out what's going on. I agree it seems silly when we're just using the gem infrastructure, but not following existing conventions seems like it may open the door to rude surprises in the future.